### PR TITLE
RW-12596 check if the target version exists first when installing GPU driver

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://precisionmedicineinitiative.atlassian.net/browse/RW-12596

Tested on BEE
```
May 15 13:52:47 saturn-dfa6ca36-18d4-42a3-ba67-fdb0dca13bc4 google_metadata_script_runner[422]: startup-script-url: + version=535.154.05
May 15 13:52:47 saturn-dfa6ca36-18d4-42a3-ba67-fdb0dca13bc4 google_metadata_script_runner[422]: startup-script-url: ++ grep 535.154.05
May 15 13:52:47 saturn-dfa6ca36-18d4-42a3-ba67-fdb0dca13bc4 google_metadata_script_runner[422]: startup-script-url: ++ cos-extensions list
May 15 13:53:17 saturn-dfa6ca36-18d4-42a3-ba67-fdb0dca13bc4 google_metadata_script_runner[422]: startup-script-url: + isAvaliable='535.154.05 [latest]'
May 15 13:53:17 saturn-dfa6ca36-18d4-42a3-ba67-fdb0dca13bc4 google_metadata_script_runner[422]: startup-script-url: + [[ -z 535.154.05 [latest] ]]
May 15 13:53:17 saturn-dfa6ca36-18d4-42a3-ba67-fdb0dca13bc4 google_metadata_script_runner[422]: startup-script-url: + cos-extensions install gpu -- --version 535.154.05
```

__Jira ticket__: https://broadworkbench.atlassian.net/browse/[ticket_number]

### What

-

### Why

-

### What to test

- 

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [ ] I validated this change 
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

File: http/src/main/resources/init-resources/gce-init.sh [hunk 1]
```diff
@@ -241,7 +241,14 @@ STEP_TIMINGS+=($(date +%s))
 
 if [ "${GPU_ENABLED}" == "true" ] ; then
   log 'Installing GPU driver...'
-  cos-extensions install gpu -- --version 535.154.05
+  version="535.154.05"
+  isAvaliable=$(cos-extensions list|grep $version)
```
Comment (LizBaldo) [inline http/src/main/resources/init-resources/gce-init.sh]: nit, `isAvailable` :)

**Review by LizBaldo — APPROVED** [review]
That logic makes sense to me, thanks 🙏

File: http/src/main/resources/init-resources/gce-init.sh [hunk 2]
```diff
@@ -241,7 +241,14 @@ STEP_TIMINGS+=($(date +%s))
...
 if [ "${GPU_ENABLED}" == "true" ] ; then
   log 'Installing GPU driver...'
-  cos-extensions install gpu -- --version 535.154.05
+  version="535.154.05"
+  isAvailable=$(cos-extensions list|grep $version)
+  if [[ -z "$isAvailable" ]]; then
+      # Install default version on the COS image
+      cos-extensions install gpu
+  else
+      cos-extensions install gpu -- --version 535.154.05
```
Comment (evrii) [inline http/src/main/resources/init-resources/gce-init.sh]: Could 535.154.05 be changed to $version here?

**Review by evrii — APPROVED** [review]
Logic makes sense to me.

File: http/src/main/resources/init-resources/gce-init.sh (hunk 2 above)
Comment (Qi77Qi) [reply http/src/main/resources/init-resources/gce-init.sh]: ah, ofc :)

**Pull request merged**